templates/chain.html

```diff
@@ -248,7 +248,12 @@ <h4>Operations</h4>
                         <option id="qgroup-{{ g }}" value="{{ g }}">{{ g }}</option>
                     {% endfor %}
                   </select>
+                  <select name="state" id="queueState" class="queueOption" style="opacity:0.5" disabled="false" onchange="checkOpformValid()">
+{#                      <option value="" disabled selected>Group</option>#}
```

Comment:
what's up with this line (see way over to the left)?

Reply:
errant comment -- now removed.